src/components/productView/productViewOpenShiftDedicated.js

```diff
@@ -96,7 +96,9 @@ ProductViewOpenShiftDedicated.defaultProps = {
 
           displayContent = translate('curiosity-graph.card-action-total', {
             context: 'coreHours',
-            total: numbro(total).format({ average: true, mantissa: 2, trimMantissa: true }).toUpperCase()
+            total:
+              (total > 999 && numbro(total).format({ average: true, mantissa: 2, trimMantissa: true }).toUpperCase()) ||
```

Comment:
optional: if there's a scenario where `999.XX` can be displayed this will attempt to apply numbro, recommend we switch to just `total >= 1000`